leases/app/controllers/MediaLeaseController.scala

```diff
@@ -38,13 +37,14 @@ object MediaLeaseController extends Controller
 
   val notFound = respondNotFound("MediaLease not found")
 
-  val appIndex = AppIndex(
-    "media-leases",
-    "Media leases service",
-    Map(),
-    List(Link("by-media-id", s"$rootUri/leases/media/{id}"))
-  )
-  def index = Authenticated { _ => respond(appIndex) }
+
+  val indexResponse = {
+    val appIndex = AppIndex("media-leases", "Media leases service", Map())
+    val indexLinks =  List(Link("by-media-id", s"$rootUri/leases/media/{id}"))
```

Comment:
I was thinking this really belongs in "Actions" not "Links", so I tried doing that, but in a surprise twist the "{ }" in the URL is incompatible with java.net.uri which is needed for Action

The solutions I found online seemed overly clunky and made the point that actually "{}" and not allowed in URIs >.>
Open to ideas 🉐